src/components/package-show.js

```diff
@@ -162,6 +166,25 @@ export default class PackageShow extends Component {
 
                   <hr />
 
+                  <label
+                    data-test-eholdings-package-details-allow-add-new-titles
+                    htmlFor="package-details-toggle-allow-add-new-titles-switch"
+                  >
+                    <h4>
+                      {packageAllowedToAddTitles
```

Comment:
This label isn't updating when I toggle. 
![qjdzmnozlo](https://user-images.githubusercontent.com/230597/36542519-a1d71d6c-17a6-11e8-9beb-4abf49f68c61.gif)

 I think it's because the label is driven off the internal component state. The other toggle switches rely on the model received in `props`. Switching this line to `model.allowKbToAddTitles` might work.

Reply:
mm, interesting. Have not seen that before. Will test it out.